console.rb

```diff
@@ -12,6 +14,14 @@
 # puts "The first apartment is at #{Apartment.first.address}."
 # puts "It has  #{Apartment.first.tenants.count} tenants."
 
-binding.pry
+# binding.pry
+
+# puts "ignore this line, it's just here so the binding.pry above works"
+puts "Please select from the following list of options"
+puts "1) View all Apartments"
+puts "2) Veiw all Tenants"
+puts "3) View all Apartments and there associated Tenants"
 
-puts "ignore this line, it's just here so the binding.pry above works"
+response = gets.chomp
+#I'm really stuck here guys.  Not sure where to move forward.  This is the work that I have completed.
+# Everything I have tried is not working.  I guess I'll just meet up with someone during office hours to complete this part.
```

Comment:
We can def go over this during our 1-on-1 this week, but I think the code you had in your PR request should work! Good effort and progress otherwise.